src/components/DropDown/DropDown.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import styled from 'styled-components'
+import { fonts } from '../../globalStyles/fonts'
+import CircleDropDown from '../../assests/arrow-circle.svg'
+
+const SelectWrapper = styled.div``
+const SelectLabel = styled.p`
+  ${fonts}
+  font-family: 'Futura Std', Arial, Helvetica, sans-serif;
+  margin: 0;
+  padding-bottom: 8px;
+  color: #979797;
+  font-size: 16px;
+  text-align: start;
+`
+const SelectStyle = styled.select`
+  ${fonts}
+  font-family: 'Futura Std', Arial, Helvetica, sans-serif;
+  max-width: 426px;
+  width: 100%;
+  height: 50px;
+  border: '1px solid #979797';
+  background-color: white;
+  border-radius: 6px;
+  outline: none;
+  padding-left: 30px;
+  color: #979797;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  background-image: url(${CircleDropDown});
+  background-repeat: no-repeat;
+  background-position-x: 96%;
+  background-position-y: center;
+  text-transform: uppercase;
+`
+
+export interface SelectProps {
+  value: string
+  placeholder: string
+  text: string
+  name: string
+  id: string
+  label: string
+}
+
+export const DropDown: React.FC<SelectProps> = ({
+  value,
+  text,
+  name,
+  id,
+  label,
+}) => {
+  return (
+    <SelectWrapper>
+      <SelectLabel>{label}</SelectLabel>
+      <SelectStyle name={name} id={id}>
+        <option value={value}>{text}</option>
```

Comment:
dropdown should handle multiple values so it should accept array of options and use map to create them